app/qml/form/FeatureForm.qml

```diff
@@ -434,6 +434,8 @@ Item {
             id: fieldLabel
 
             text: Name
+            height: visible ? paintedHeight : 0
```

Comment:
Is it necessary to set the height to 0?

Reply:
I think yes, on my iOS it was taking (white) place